plugin/gcp/cloudRunDir.go

```diff
@@ -0,0 +1,62 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/puppetlabs/wash/plugin"
+	"google.golang.org/api/option"
+	"google.golang.org/api/run/v1"
+)
+
+type cloudRunProjectAPIService struct {
+	*run.APIService
+	projectID string
+	// We need to pass this around to access cloud function logs
+	client *http.Client
+}
+
+type cloudRunDir struct {
+	plugin.EntryBase
+	apiService cloudRunProjectAPIService
+}
+
+func newCloudRunDir(ctx context.Context, client *http.Client, projID string) (*cloudRunDir, error) {
+	svc, err := run.NewService(context.Background(), option.WithHTTPClient(client))
+	if err != nil {
+		return nil, err
+	}
+	cr := &cloudRunDir{
+		EntryBase:  plugin.NewEntry("cloud_run"),
```

Comment:
I guess `run` might be a little too generic. Would be symmetrical with `Cloud Storage` though.

Reply:
I went with what's shown in the console. There, `Cloud Storage` is `Storage` while `Cloud Run` is `Cloud Run`.